policyengine_us/variables/gov/irs/income/taxable_income/adjusted_gross_income/irs_gross_income/social_security/taxable_social_security_tier_2.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class taxable_social_security_tier_2(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    label = "Taxable Social Security (tier 2)"
+    documentation = "Taxable Social Security from 50-85% taxation tier, credited to Medicare HI trust fund"
+    unit = USD
+    reference = "https://www.law.cornell.edu/uscode/text/26/86#a_2"
+    subtracts = [
+        "tax_unit_taxable_social_security",
+        "taxable_social_security_tier_1",
+    ]
```

Comment:
Does this work? Should it not be adds `tax_unit_taxable_social_security` and subtracts `taxable_social_security_tier_1`